src/main/java/io/curity/authenticator/netid/PollingAuthenticatorConstants.java

```diff
@@ -66,6 +66,12 @@
          */
         public static final String UNKNOWN_PERSONAL_NUMBER = "cur3";
 
+        /**
+         * Blocks authentication requests during the specified request block time if a user has an active authentication request
+         * and a new request is issued. Request block time is set in server's settings.xml.
+         */
+        public static final String USER_BLOCKED="cur4";
```

Comment:
I would use something like `user_blocked` instead. The old pattern is just confusing.

Reply:
updated all of the end user message keys to the suggested format.